### DISCUSSION TITLE
Dodanie zbailansowanej energii dla bieżącego roku i kilka aktualizacji w Readme

### DISCUSSION THREAD

**Discussion by Jordan87** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#143] [Ideas]
Czesc

Oczywiscie juz teraz jestem mega wdzieczny za caly dodatek ale mam pomysl na drobne usprawnienie :)
Jak wiemy po przekroczeniu limitu 2000kWh (lub wiecej przy karcie duzej rocziny lub rolnictwie) placimy wiecej.
Majac fotowoltaike interesuje nas tylko energia zbilansowana godzinowo wiec milo bylo by miec ilosc kWh pobranej w w danym roku po bilansowaniu, aktualnie tego brakuje w danych. (energi wyslana w danym roku chyba jest zbedna ale moze i dla tego ktos by znalazl zastosowanie). :)

Co do aktualizacji readme dodal bym tam krotka informacje o tym ze aktywowanie "Włącz konfigurowalne sensory" i "Włącz sensory zakresu 12 miesięcy" tworza nowe encje. I co one zaweiracją. Aktualnie nie ma wzmianki w readme o konfigurowalnych.
Opcja "Włącz sensory zakresu 12 miesięcy" zawsze trzyma ostatnie 365dni (usuwa co dzien jeden dzien ze statystyk)?
![image](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/assets/10414918/28df0475-238e-47b7-9511-59754813e23c)

Dzięki za całą twoja pracę!

**Comment by PiotrMachowski** [comment]
> Majac fotowoltaike interesuje nas tylko energia zbilansowana godzinowo wiec milo bylo by miec ilosc kWh pobranej w w danym roku po bilansowaniu,

Do tej pory specjalnie tego nie dodałem, żeby nie pobierać danych godzinowo z całego roku (dzień po dniu), ale ostatnio dodałem cache'owanie odpowiedzi z API, więc chyba mogę takie coś dodać.

W README brakuje kilku rzeczy, może się zbiorę i je dodam 😉 

> Opcja "Włącz sensory zakresu 12 miesięcy" zawsze trzyma ostatnie 365dni (usuwa co dzien jeden dzien ze statystyk)?

Tak (w szczególnych przypadkach 366). Tylko bym to bardziej nazwał to zakresem, a nie statystykami. Statystyki to pozycje `tauron_importer` w panelu Energia

**Reply by PiotrMachowski** [reply]
@Jordan87 Funkcje dodałem w [v2.5.0](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus/releases/tag/v2.5.0)

**Reply by Jordan87** [reply]
Dzięki, przetestuje :)

**Reply by PiotrMachowski** [reply]
Ale za kolejne pomysły będzie ban :D

**Reply by Jordan87** [reply]
Nie mozna tego robic ludziom w kwiecie kreatywności :D
A jak wymysle perpeetum mobile i bede sie chcial tym podzielić?

**Reply by PiotrMachowski** [reply]
Teraz powinienem się skupić w końcu na innych customach ;)

**Reply by Jordan87** [reply]
To żeby nei zatruwać ci życia kolejnymi pomysłami wysłałem napiwek na drugą kawę ;)

**Reply by PiotrMachowski** [reply]
Dzięki, po zarwanej nad Tauronem nocy kawa na pewno nie zaszkodzi :D
